### PR TITLE
added interface for lecturer printing, will proceed to backend

### PR DESCRIPTION
**Pull request by ziqiang2344**

### CODE REVIEW

**Comment by ziqiang2344** [comment]
added interface, about to proceed to backend

File: src/sample/downloadController.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
   private TextField textfield;
   @FXML
   private Text pdfMessage;
-
+
```
Comment (Michelle-Lohwt) [inline src/sample/downloadController.java]: extra spaces

File: src/sample/downloadController.java [hunk 2]
```diff
@@ -119,7 +119,8 @@ public void handleButtonAction(ActionEvent event) throws FileNotFoundException,I
     }
 
   }
-  
+
+
```
Comment (Michelle-Lohwt) [inline src/sample/downloadController.java:279]: extra indentation

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Can you show StuListDownload and TeachCourseDownload in the picture?

And please check the comments

**Comment by ziqiang2344** [comment]
image
The lecturer printing course photo

**Comment by ziqiang2344** [comment]
will proceed to print student list next

**Comment by ziqiang2344** [comment]
image
The student list look like this. You'll see the student ID have extra 1 preceding the original studentID but no worries, that's just because the original student name is "Student Name 1" and the 1 is appended with the student ID. This case won't happen if we inserted the real student name, cuz name won't contain number.

**Pull request merged**